cluster_mgr_setup.py

```diff
@@ -25,7 +25,8 @@
     test_suite='metaswitch.clearwater.cluster_manager.test',
     install_requires=[
         "clearwater_etcd_shared",
-        "metaswitchcommon"],
+        "metaswitchcommon",
+        "python-consul==0.7.0"],
```

Comment:
So will Clearwater need new dependencies, even though they're only used on DM?  Is Ellie OK with that?  (I don't imagine the impact is major - and fwiw third-party dependency stuff I'm sweeping up under the orchestration umbrella.)